client/components/comments/comment.tsx

```diff
@@ -0,0 +1,25 @@
+// src/components/Comment.tsx
+import React from 'react';
+import { motion } from 'framer-motion';
+import { CommentType } from '../../../shared/types';
+
+interface CommentProps {
+  comment: CommentType;
+}
+
+const Comment: React.FC<CommentProps> = ({ comment }) => {
+  return (
+    <motion.div
+      className="bg-white text-black p-3 rounded-md shadow-md mb-2 absolute bottom-0"
+      initial={{ opacity: 0, y: 0 }}
+      animate={{ opacity: 1, y: -150, transition: { duration: 2 } }}
```

Comment:
the animation looks like it fades and moves up to the middle -- probably need to check with the prof if that's ok?
i was thinking maybe the comments fully visible at the very bottom of the screen similar to the [figma prototype](https://www.figma.com/proto/jnVHKQftbSRXTpkuRfMobp/UI-Mockups?node-id=61-606&t=ks8edYBEqpTgwUuM-1&scaling=scale-down&content-scaling=fixed&page-id=1%3A2&starting-point-node-id=61%3A606&show-proto-sidebar=1) but with all the nice fade in animations?
(we can address this in a future pr!)

Reply:
will take a look at this in a later pr :)